docs/FAQ/gui-faq-en.md

```diff
@@ -4,163 +4,178 @@ Zonemaster
+- Your domain name is not reachable from public Internet.
+- Zonemaster can only test what is called a DNS zone (e.g. 'zonemaster.net') and not host names (e.g. 'www.zonemaster.net')
+- There is a 10 minutes protection between consecutive tests for a given domain name (with same test parameters).
+  Running a test within that window will instead show the last available test for that domain name (and parameters).
+- You have misspelled your domain name.
 
 <a name="q11"></a>
 #### 11. What kind of queries does Zonemaster generate?
-Zonemaster send multiple DNS queries to the name servers hosting the domain name and
-also to the name servers hosting the parent zone of the domain name.
+Zonemaster send multiple DNS queries to the name servers hosting the domain name being tested and
+also to the name servers hosting the parent zone of that domain name.
 
-To get a full
-view of what queries and results are generated you can run the
-CLI interface (and in order to run the CLI interface you need to download the
-complete package and install it) and select full output.
-The output from the CLI tool is quite heavily technical
-so unless you are into bits and bytes you might want to skip this step.
+The GUI interface of Zonemaster does not show any queries sent.
+Only the CLI interface can. If you want to see such queries, you will have to locally install
+a minimally working Zonemaster instance with both the Engine and CLI components.
+Queries sent can be shown using the 'DEBUG' level option.
+Fair warning, the output from the CLI can be quite heavy.
 
 <a name="q12"></a>
 #### 12. What is an undelegated domain test?
-An undelegated domain test is a test performed on a domain that may, or may not,
-be fully published in the DNS. This can be quite useful if one is going to move
-one's domain from one registrar to another,
-e.g., move zone example.com from the nameserver
-"ns.example.com" to the nameserver "ns.example.org". In this scenario one could perform
-an undelegated domain test providing the zone (example.com) and the nameserver you are moving to
-(ns.example.org) *before* you move your domain.
-When the results of the test are colour coded in green one can be fairly certain
-that the domain's new location is working well. However there
-might still be other problems in the zone data itself that this test is unaware of.
+An undelegated domain test is a test performed on a domain name that may, or may not,
+be fully published in the DNS.
+This can be quite useful if one is going to migrate one's domain from one registrar to another,
+e.g., migrate zone 'example.com' from the name server 'ns.example.com' to the name server 'ns.example.org'.
+In this scenario one could perform an undelegated domain test providing the zone ('example.com')
+and the name server you are migrating to ('ns.example.org') *before* you migrate your domain.
+When the results of the test are color-coded "green" one can be fairly certain that the
```

Comment:
I suggest:
`When the results of the test are color-coded "green" one can be` --> `When the results of the test show errors or warnings one can be`

Reply:
Updated